### PR TITLE
Make symfony/console a regular dependency and not just a dev dependency

### PR DESCRIPTION
**Pull request by bertramakers**
### Fixed
 
- Fixed broken `./bin/app.php` script when using `composer install --no-dev`

### CODE REVIEW

**Review by LucWollants — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
         "ramsey/uuid": "^3.9",
         "sentry/sdk": "^2.2",
         "slim/psr7": "^0.6",
+        "symfony/console": "^4.4.20",
```
Comment (jdrieghe) [inline composer.json:35]: Why is this set specifically to `^4.4.20`?
This way it will never update to a potential `4.5`

Comment (bertramakers) [reply composer.json:35]: Ah yes. I just took the version we had in the lock file.

Comment (bertramakers) [reply composer.json:35]: https://github.com/cultuurnet/udb3-search-service/pull/122

Comment (LucWollants) [reply composer.json:35]: I will also try to remember this 😊

**Pull request merged**